### PR TITLE
[WIP] org.kde.rkward

### PR DESCRIPTION
**Pull request by tfry-git**
@travier

### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
--> filesystem=host: RKWard is a frontend to R. R is a statistics language an environment with literally thousands of add-on-packages. These are downloadable as sources and need a compiler and arbitrary libraries to install. Packaging all that is practically impossible, and out-of-scope.
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
--> community owned (KDE.org)
- [NA] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.kde.rkward.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    "id": "org.kde.rkward",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "5.15-21.08",
+    "command": "rkward",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
```
Comment (barthalion) [inline org.kde.rkward.json:11]:
```suggestion
```

File: org.kde.rkward.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+    "id": "org.kde.rkward",
+    "branch": "master",
```
Comment (barthalion) [inline org.kde.rkward.json:3]:
```suggestion
```

**Comment by barthalion** [comment]
cc @flathub/kde

**Comment by barthalion** [comment]
bot, build org.kde.rkward

File: org.kde.rkward.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                        "type": "anitya",
+                        "project-id": 236337,
+                        "stable-only": true,
+                        "url-template": "https://download.kde.org/stable/release-service/$version/src/kate-$version.tar.xz"
+                    }
+                }
+            ]
+        },
+        {
+            "buildsystem": "cmake",
```
Comment (hfiguiere) [inline org.kde.rkward.json:57]:
```suggestion
            "buildsystem": "cmake-ninja",
```

(unless it doesn't work)

File: org.kde.rkward.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    "command": "rkward",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host"
```
Comment (hfiguiere) [inline org.kde.rkward.json:18]: I don't think this will grant you access to what you want. You won't have access to whatever the distro installs.

**Comment by hfiguiere** [comment]
@kevinsmia1939 I think there is overlap with your on #2597, with similar problems.

File: org.kde.rkward.json (hunk 1 above)
Comment (tfry-git) [reply org.kde.rkward.json]: Copy and paste leftovers. Removed.

File: org.kde.rkward.json (hunk 4 above)
Comment (tfry-git) [reply org.kde.rkward.json:15]: Sigh... I see what you mean (I had tested that I was able to load packages outside the app filesystem, but not compiling new packages.)

I have a plan (of sorts) for working around this, but it won't be something to patch in a few minutes and/or lines, so putting this on hold for now ([WIP] tag added).

**Comment by tfry-git** [comment]
Abandoning for now.

**Pull request closed without merging**